next.config.js

```diff
@@ -3,4 +3,11 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
   enabled: process.env.ANALYZE === "true",
 });
 
-module.exports = withBundleAnalyzer(withTM({}));
+module.exports = withBundleAnalyzer(
+  withTM({
+    reactStrictMode: true,
+    images: {
+      domains: [`${process.env.IMAGE_BUCKET}.s3.amazonaws.com`],
```

Comment:
With automatic image optimization you have to allowlist the domains the images come from that you want to cache.